packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -106,6 +107,11 @@ export interface IDropdownProps extends React.Props<Dropdown> {
   required?: boolean;
 
   /**
+   * Custom properties for Dropdown's Callout.
+   */
+  calloutProperties?: ICalloutProps;

```

Comment:
This is creating an impression that we are supporting all callout properties and we may get issues raised in future. I think we should just name take what is required `dropContainerClass: string` to set the right expectations.